client/components/checkbox/README.md

```diff
@@ -0,0 +1,32 @@
+Checkbox
+========
+
+Checkbox is a React component used to render a checkbox input field. It is essentially an enhanced version of `<input type="checkbox" />`, using Gridicon as check mark.
+
+## Usage
+
+Refer to the following code snippet for a typical usage example:
+
+```jsx
+<Checkbox
+	checked={ this.state.showSubtitles }
+	onChange={ this._toggleShowSubtitles }>
```

Comment:
Since [our style guide](https://github.com/Automattic/wp-calypso/blob/master/docs/coding-guidelines/javascript.md) tells us to avoid prefixing a function name with `_`, we probably shouldn't use that in an Readme example :wink: